cea/analysis/uncertainty/Uncertainty_parameters.py

```diff
@@ -0,0 +1,266 @@
+" This file is used to generate various scenarios which involve uncertainty. Go through README.md"
+
+__author__ = "Sreepathi Bhargava Krishna"
+__copyright__ = "Copyright 2015, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Sreepathi Bhargava Krishna"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Daren Thomas"
+__email__ = "cea@arch.ethz.ch"
+__status__ = "Production"
+
+def run_as_script(scenario_path):
+    import pandas as pd
+    import cea.inputlocator
+    import numpy as np
+
+    # Importing the excel sheet containing the uncertainty distributions
+    locator = cea.inputlocator.InputLocator(scenario_path)
+    uncertainty_db = pd.read_excel(locator.get_uncertainty_db(), 'ECONOMIC')
+
+    # Fetching the distribution type and the corresponding variables from the excel sheet
+    HP_n_distribution = uncertainty_db[uncertainty_db['name'] == 'HP_n']['distribution'].max()  # lifetime [years] distribution 20
```

Comment:
all this is hardcoding material, keep the run_as_script short and create functions if necessary.  follow the cea.demand_main.run_as_script as an example